src/ir/item.rs

```diff
@@ -78,6 +78,20 @@ impl ItemCanonicalPath for ItemId {
     }
 }
 
+/// An item is the base of the bindgen representation, it can be either a
+/// module, a type, a function, or a variable (see `ItemKind` for more
+/// information).
+///
+/// Items form a tree, and each item only stores the id of the parent.
```

Comment:
They're actually a graph because of the de-duplication with `ItemId`, right? Maybe worth clarifying.